tox.ini

```diff
@@ -19,7 +19,14 @@ passenv = CI TRAVIS TRAVIS_*
 deps = -r{toxinidir}/requirements/testing.txt
 changedir = {toxinidir}/readthedocs
 commands =
-    pytest --cov-report= --cov-config {toxinidir}/.coveragerc --cov=. --suppress-no-test-exit-code {posargs:{env:TOX_POSARGS:-m 'not search'}}
+    /bin/sh -c '\
+        export DJANGO_SETTINGS_MODULE=readthedocs.settings.test; \
+        pytest --cov-report= --cov-config {toxinidir}/.coveragerc --cov=. --suppress-no-test-exit-code -m "not proxito" {posargs:{env:TOX_POSARGS:-m "not search and not proxito"}}'
```

Comment:
`-m "not proxito" {posargs:{env:TOX_POSARGS:-m "not search and not proxito"}}`

It's hard to make everyone happy :D. So, `-m` doesn't combine the multiple `-m`, but it takes the last `-m option`, so search tests and proxito tests aren't run by default. And proxito tests won't run for people that wants to run search tests.